CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+The development tracker for odbc_fdw is on GitHub:
+http://github.com/cartodb/odbc_fdw
+
+---
+
+1. [General](#general)
+2. [Bugs and Features](#bugs-and-features)
+3. [Submitting contributions](#submitting-contributions)
+
+---
+
+# General
+
+Every new feature (as well as bugfixes) should come through a [github pull requests](https://help.github.com/articles/using-pull-requests) and should have test cases for it.
+
+Go [here](https://github.com/CartoDB/odbc_fdw/tree/master/test#connectors-integration-tests) to learn how the test framework works and how to add new tests to the project.
+
+# Bugs and features
+
+Before you start working on a new feature, we encourage you to create an issue in github explaining what are you going to do in order to help you and especially to avoid duplicate work.
```

Comment:
> Before you start working on a new feature

"Before you start working on a new **bug or feature**..."

> we encourage you to create an issue

I'd say "we encourage you to first look up the open issues list to check there's no duplicate. If your issue is not described there, please create one..."

> what are you going

"what you are going to do" instead